wagtail/images/tests/test_image_operations.py

```diff
@@ -580,7 +580,7 @@ def test_default_quality(self):
         save.assert_called_with(f, 'JPEG', quality=85, optimize=True, progressive=True)
 
     def test_jpeg_quality_filter(self):
-        fil = Filter(spec='width-400|jpegquality-40')
+        fil = Filter(spec='width-400|quality-40')
```

Comment:
Is it possible that someone would want to specify different qualities for JPEG/WEBP?

Reply:
Yes, it is possible. But keep in mind that you set this setting for a specific image and it is either JPEG or WebP. You would never use both of them at the same time.

Reply:
That's the case if the template is only used in one place on the site. But if the template is used multiple times, you would have different images using the same filter.

It's probably fine the way it is though as long as the scale is always 1 - 100. But I wonder why the defaults are different for the two formats?

Reply:
JPEG and WebP quality visually differs with the same quality setting. That's why it seemed more practical to choose different default settings. I agree with you. I will specify different filters for JPEG and WebP.

Reply:
Thanks!